lib/buffer.js

```diff
@@ -102,7 +105,7 @@ function Buffer(arg, encodingOrOffset, length) {
         'If encoding is specified then the first argument must be a string'
       );
     }
-    return Buffer.allocUnsafe(arg);
+    return Buffer.allocUnsafe(arg).fill(randomFill);
```

Comment:
Yep, it is. I went with this approach here because it requires the least amount of code. It is more performant to do a `memset` in the Allocator inside `node.cc`, but doing so is a bit more intrusive and introduces a branch that impacts all Allocations (albeit, an extremely minor one).

I maintain that the best thing to do would be to just use `calloc` and zero-fill instead.